<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
    <item>
     <widget class="QLabel" name="PainterWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1500</width>
        <height>900</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1500</width>
        <height>900</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>900</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>900</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>720</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Алгоритм</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="DDACheck">
            <property name="text">
             <string>ЦДА</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="BresenhamRealCheck">
            <property name="text">
             <string>Алгоритм Брезенхэма (действительные)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="BresenhamIntegerCheck">
            <property name="text">
             <string>Алгоритм Брезенхэма (целочисленные)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="BresenhamNoStepsCheck">
            <property name="text">
             <string>Алгоритм Брезенхэма с устранением ступенчатости)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="WuCheck">
            <property name="text">
             <string>Алгоритм Ву</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="LibraryCheck">
            <property name="text">
             <string>Функция библиотеки QT</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Цвета</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="LineColorButton">
            <property name="text">
             <string>Цвет линий</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="LineColorLabel">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="BackgroundColorButton">
            <property name="text">
             <string>Цвет фона</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="BackgroundColorLabel">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Построение отрезка</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Точка 2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="Point2XLabel">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="Point2XInput">
               <property name="minimum">
                <number>-900000000</number>
               </property>
               <property name="maximum">
                <number>900000000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="Point2YLabel">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="Point2YInput">
               <property name="minimum">
                <number>-900000000</number>
               </property>
               <property name="maximum">
                <number>900000000</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="DrawSegmentButton">
            <property name="text">
             <string>Построить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Точка 1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="Point1XLabel">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="Point1XInput">
               <property name="minimum">
                <number>-900000000</number>
               </property>
               <property name="maximum">
                <number>900000000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="Point1YLabel">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="Point1YInput">
               <property name="minimum">
                <number>-900000000</number>
               </property>
               <property name="maximum">
                <number>900000000</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Построение спектра</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Угол</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="RangeAngleInput">
            <property name="minimum">
             <number>-9000000</number>
            </property>
            <property name="maximum">
             <number>9000000</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Радиус</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="RangeRadiusInput">
            <property name="minimum">
             <number>-9000000</number>
            </property>
            <property name="maximum">
             <number>9000000</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="DrawRangeButton">
            <property name="text">
             <string>Построить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearScreenButton">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ResearchTimeButton">
         <property name="text">
          <string>Время</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ResearchStepsButton">
         <property name="text">
          <string>Ступенчатость</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
